k8s/client.go

```diff
@@ -1084,6 +1085,39 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+func (c *Client) GetRESTClientGetter() (*genericclioptions.RESTClientGetter, error) {
+	if c.RESTClientGetter == nil {
+		return nil, fmt.Errorf("unable to get the RESTClientGetter")
+	}
+	return &c.RESTClientGetter, nil
+}
+
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string, namespace string) (string, error) {
+	restClientGetter, err := c.GetRESTClientGetter()
+	if err != nil {
+		return "", fmt.Errorf("unable to get the RESTClientGetter from the client")
```

Comment:
If we do keep this function, you can `return "", err` here, since the error messages are essentially identical.

Reply:
With the above section, you can remove this block